recipes/nodejs_from_binary.rb

```diff
@@ -57,5 +57,6 @@
   version node['nodejs']['version']
   checksum checksum
   has_binaries binaries
+  append_env_path node['nodejs']['binary']['append_env_path']
```

Comment:
I had this exact same issue. `append_env_path` adds the `/usr/local/nodejs-binary/bin` to `PATH`. `has_binaries` just symlinks the node and npm binaries into `/usr/local/bin`

This is a good fix  - though it brings up a bug in ark on Mac where they use root:root to add the profile.d script, and that Mac's /etc/profile doesn't use profile.d by default unless you hack it. May want to limit it to linux platform until ark is fixed to use root:wheel on mac.